Classes/Controllers/OpenRecentController.m

```diff
@@ -146,8 +145,8 @@ - (NSInteger)numberOfRowsInTableView:(NSTableView *)aTableView
 }
 
 - (IBAction)changeSelection:(id) sender {
-	int i = [resultViewer selectedRow];
-	if(i >= 0 && i < [currentResults count])
+	NSUInteger i = resultViewer.selectedRow;
+	if(i < currentResults.count)
```

Comment:
Yeah, `-Wsign-conversion` catches this. (_Implicit Signedness Conversions_ in Xcode.) Turning it on generates a good number of new warnings.